### PR TITLE
Allow unescaped json output

### PR DESCRIPTION
**Pull request by nagypeterjob**
#1430 

Added `toUnescapedJSON` and `toUnescapedJSONPretty` functions (and tests) to be able to generate JSON without HTML escaping the resulting string. See Issue for more details.

I have also updated README.

### CODE REVIEW

**Review by eikenb — APPROVED** [review]
This looks good. Going to delay merging it just slightly as I'll need to update the documentation refactor branch when I do and I can't do that right this moment.

**Comment by nagypeterjob** [comment]
Thanks in advance @eikenb

File: README.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ this functionality might prove useful.
       - [timestamp](#timestamp)
       - [toJSON](#tojson)
       - [toJSONPretty](#tojsonpretty)
+      - [toUnescapedJSON](#tounescapedjson)
```
Comment (eikenb) [inline README.md:96]: This documentation has moved while this has been waiting (sorry). If you have a chance and wouldn't mind @nagypeterjob, please consider updating this PR with the documentation moved into docs/configuration.md. If you don't have time just say so, I can make the update if necessary.

Thanks.

Comment (nagypeterjob) [reply README.md:96]: It would be really nice of you if you could do the change @eikenb. Also, I am not sure how to resolve the conflicts.

Comment (eikenb) [reply README.md:96]: I'll be happy to handle the doc changes and I can fix the conflicts while I'm at it.

**Pull request merged**